.github/workflows/test.yml

```diff
@@ -96,6 +95,15 @@ jobs:
       - name: fetch ssl certificate after tests (linux, os x)
         if: matrix.runner != 'windows-latest'
         run: echo | openssl s_client -showcerts -servername 'api.vercel.com' -connect 76.76.21.21:443
+      - name: install datadog-ci
```

Comment:
Datadog also has a [GitHub Action](https://github.com/DataDog/junit-upload-github-action) for this but it's just basically just taking the DD cli interface and exposing that through specific yaml keys. IMO, not worth adding.